Sources/FeaturevisorSDK/Emitter.swift

```diff
@@ -5,7 +5,7 @@ public enum EventName {
     case activation
 }
 
-public typealias Listener = (@escaping (Any...) -> Void) -> Void
+public typealias Listener = (Any...) -> Void
```

Comment:
Will be breaking change, not saying it's wrong. Just we need to be aware of this.